From 0d55c87e7e10ae68739adddbed3d058bf828e4d9 Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Mon, 22 Oct 2018 11:32:26 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808-evb-v10: fix aloop card index to
 7

This patch fix the aloop card index to 7, and leave the card 0
place for default hw snd card.

Change-Id: I767d3d61b1ecb09b813d7bf81b99298cd30a3969
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
index fdc09ac63f88..a6529c604215 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
@@ -12,7 +12,7 @@
 	compatible = "rockchip,rk1808-evb-v10", "rockchip,rk1808";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff550000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait swiotlb=1 kpti=0";
+		bootargs = "earlycon=uart8250,mmio32,0xff550000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait swiotlb=1 kpti=0 snd_aloop.index=7";
 	};
 
 	vad-sound {
-- 
2.35.3

